From khali@linux-fr.org Fri Sep 23 00:04:16 2005
Date: Thu, 22 Sep 2005 21:47:58 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH] (1/7) i2c-viapro: New maintainer
Message-Id: <20050922214758.3052c5e1.khali@linux-fr.org>

I am taking over the maintenance of the i2c-viapro SMBus driver.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 MAINTAINERS |    6 ++++++
 1 file changed, 6 insertions(+)

--- gregkh-2.6.orig/MAINTAINERS	2005-09-28 15:53:19.000000000 -0700
+++ gregkh-2.6/MAINTAINERS	2005-09-28 15:54:04.000000000 -0700
@@ -2704,6 +2704,12 @@
 M:	rl@hellgate.ch
 S:	Maintained
 
+VIAPRO SMBUS DRIVER
+P:	Jean Delvare
+M:	khali@linux-fr.org
+L:	lm-sensors@lm-sensors.org
+S:	Maintained
+
 UCLINUX (AND M68KNOMMU)
 P:	Greg Ungerer
 M:	gerg@uclinux.org
